src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -20,15 +20,10 @@
 import azure.cli.core.azlogging as azlogging
 from azure.cli.core.util import CLIError, todict, shell_safe_json_parse
 from azure.cli.core.profiles import ResourceType
+from msrestazure.tools import parse_resource_id, is_valid_resource_id
```

Comment:
I think for perf. we recommend loading msrestazure only when needed. i.e. in the methods you use it in.